python/excludeVMs/excludeVMs.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
received error `/usr/bin/env: ‘python’: No such file or directory` 
probably since python is python2 and that is EOL. my system calls it python3 these days.